screens/Library.jsx

```diff
@@ -10,18 +12,22 @@ function Library({ route }) {
         blurRadius: 1,
     };
     return (
+       
         <Main name={name} coverImage={coverImage}>
-            <View style={styles.center}>{/* No Content */}</View>
+            <View  style = {styles.container}>
```

Comment:
Add scroll view here to make it scrollable?

Reply:
Yes, and make the open hours section not scrollable.